http://sourceware.org/ml/gdb-cvs/2012-04/msg00184.html

### src/gdb/doc/ChangeLog	2012/04/23 17:20:56	1.1300
### src/gdb/doc/ChangeLog	2012/04/23 17:26:00	1.1301
## -3,6 +3,9 @@
 	* gdb.texinfo (Auto-loading safe path): Remove trailing [@dots{}].
 	Three times.
 
+	* gdb.texinfo (Auto-loading safe path): Add trailing @dots{}.
+	Three times.
+
 2012-04-22  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* gdb.texinfo (Auto-loading safe path): Replace @itemize @bullet
--- src/gdb/doc/gdb.texinfo	2012/04/23 17:20:56	1.947
+++ src/gdb/doc/gdb.texinfo	2012/04/23 17:26:01	1.948
@@ -21142,11 +21142,11 @@
 You have to specify also any existing directories displayed by
 by @samp{show auto-load safe-path} (such as @samp{/usr:/bin} in this example).
 
-@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb"}
+@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb" @dots{}}
 Specify this directory as in the previous case but just for a single
 @value{GDBN} session.
 
-@item @kbd{gdb -iex "set auto-load safe-path"}
+@item @kbd{gdb -iex "set auto-load safe-path" @dots{}}
 Disable auto-loading safety for a single @value{GDBN} session.
 This assumes all the files you debug during this @value{GDBN} session will come
 from trusted sources.
@@ -21161,7 +21161,7 @@
 also suppresses any such warning messages:
 
 @table @asis
-@item @kbd{gdb -iex "set auto-load no"}
+@item @kbd{gdb -iex "set auto-load no" @dots{}}
 You can use @value{GDBN} command-line option for a single @value{GDBN} session.
 
 @item @file{~/.gdbinit}: @samp{set auto-load no}
